Urban Synergy Unleashed/common/buildings/yMoG_USU_railways.txt

```diff
@@ -136,18 +136,13 @@
 			add = 10000
 		}
 		
-		if = { # Get them started at least...
-			limit = {
-				NOR = {
-					has_building = building_railway
-					any_scope_building = {
-						is_building_type = building_railway
-						is_under_construction = yes
-					}
-				}
-			}
-			add = 5000
-		}
+		# AI shouldn't go around building railways before it needs them
+		#if = { # Get them started at least...
+		#	limit = {
+		#		NOT = { exists = b:building_railway }
+		#	}
+		#	add = 2000
+		#}
```

Comment:
It is worth noting that in previous versions, the AI often wouldn't create the demand needed to trigger railway buildings in the first place. We should keep an eye out for this behaviour recurring.